clients/drcachesim/scheduler/scheduler.cpp

```diff
@@ -1515,7 +1565,8 @@ scheduler_tmpl_t<RecordType, ReaderType>::get_tid(output_ordinal_t output)
     int index = outputs_[output].cur_input;
     if (index < 0)
         return -1;
-    if (inputs_[index].is_combined_stream())
+    if (inputs_[index].is_combined_stream() ||
+        TESTANY(OFFLINE_FILE_TYPE_CORE_SHARDED, inputs_[index].reader->get_filetype()))
```

Comment:
Should is_combined_stream be modified to return true for core-sharded-on-disk?

Reply:
No, I looked into this before and the other uses of is_combined_stream do not apply to core-sharded.

Reply:
It should probably be documented then. A core-sharded stream does seem "combined" in a way so some users may expect it to return true.

Reply:
(It's not a public interface so it wouldn't be users looking at it, but developers.)  Added clarifying documentation.